README.md

```diff
@@ -67,7 +67,7 @@ rest_cherrypy:
 
 **Note: With this configuration, the user has access to all salt modules available, maybe this is not what you want**
 
-Please read the [Permissions](docs/PERMISSIONS.md) page for more information.
+Please read the [Permissions](guide/PERMISSIONS.md) page for more information.
 
 
 ## Authentication
```

Comment:
I am a little confused, so in that diff up above, would you want me to remove that line (line 70) all together since its now in the mdbook docs? Because everything in your `README` is in the mdbook docs. I can remove all the text in the `README` if you would like, but keep the about `SaltGUI` section.

Reply:
this was a generic remark about the README.md file.
it has nothing to do with the nearby lines.

we have to find a proper separation between the documentation in README.md and the mdbook docs.
so that double-maintenance is avoided.